RiotSharp.Test/RiotApiTest.cs

```diff
@@ -92,35 +92,7 @@ public void GetChampionRotationAsync_Test()
 
         [TestMethod]
         [TestCategory("RiotApi"), TestCategory("Async")]
-        public void GetLeagueEntriesBySummonerAsync_Test()
-        {
-            EnsureCredibility(() =>
-            {
-                // TODO: Properly implement encrypted SummonerId tests
-                return;
-                var leagues = Api.League.GetLeagueEntriesBySummonerAsync(RiotApiTestBase.SummonersRegion, RiotApiTestBase.SummonerIds.FirstOrDefault());
-
-                Assert.IsTrue(leagues.Result.Count > 0);
-            });
-        }
-
-        [TestMethod]
-        [TestCategory("RiotApi"), TestCategory("Async")]
-        public void GetLeagueByIdAsync_Test()
-        {
-            EnsureCredibility(() =>
-            {
-                // TODO: Properly implement League id test
-                return;
-                var leagues = Api.League.GetLeagueByIdAsync(RiotApiTestBase.SummonersRegion, "LEAGUE-ID-HERE");
-
-                Assert.IsTrue(leagues.Result.Queue != null);
-            });
-        }
```

Comment:
why was this removed?

Reply:
I removed two tests, because they both had TODO's with "Properly implement"

So i just assumed they where not in use or will be fixed because the tests with TODO's seem very old.

Please let me know if you would like them added back in :)

Reply:
let's leave them in :+1:

Reply:
I have added them back in and renamed them 👍